### PR TITLE
Hw 6

### PR DESCRIPTION
**Pull request by amblount**

### CODE REVIEW

**Comment by tgaff** [comment]
@chylaudes we dealt with some git stuff after class on monday

File: AliviaBlount-amblount/hw_6/rise_of_robot.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+# this method creates new instances of the robot class
+#instance variables always start with an @
+#instance variables are available throughout the class
+#@name is an instance variable
+  def initialize(name, type, origin)
+    @name = name+"!"
+    @type = type
+    @origin = origin
+  end
+
```
Comment (chylaudes) [inline AliviaBlount-amblount/hw_6/rise_of_robot.rb]: Just for best practice sake, move the def initialize block as the first method you're defining after class Robot

File: AliviaBlount-amblount/hw_6/thermostat.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  attr_reader :target
+
+  def initialize(temp, target)
+  @degrees = temp
+  @target_temp = target
+  end
+
+  def self.direct_temperature(all_temps, target_temp)
+    all_temps.each do |temp|
+      reading = Thermostat.new(temp, target_temp)
```
Comment (chylaudes) [inline AliviaBlount-amblount/hw_6/thermostat.rb]: It is actually `Temperature.new(temp, target_temp)`

File: AliviaBlount-amblount/hw_6/thermostat.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      reading.change_temp
```
Comment (chylaudes) [inline AliviaBlount-amblount/hw_6/thermostat.rb]: The method for change_temp actually needs to take in to two parameters the `temp` and `target_temp`
```
 reading.change_temp(temp, target_temp)
```

**Comment by chylaudes** [comment]
Hi! So your robot.rb was great!  There are some errors when I ran through the code and it has to do with the class missing an `end` in a couple of pages.  Comb through the code and makes sure you're closing the class Temperature and your methods correctly.  Feel free to ping me any questions. We can definitely go over class methods and instance methods too!

File: AliviaBlount-amblount/hw_6/thermostat.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
+#read temp
+#set target temp
+#detect temp
+  #A. new instance of temp
+  #B. get desired temperature
+  #C. turn-off or turn-on
+
+class temperature
+
```
Comment (chylaudes) [inline AliviaBlount-amblount/hw_6/thermostat.rb]: To create a class you must capitalize
```
class Temperature
```

**Pull request merged**